code/routers/country_page/country.js

```diff
@@ -99,28 +99,196 @@ router.get('/get-country-info', (req, res) => {
 /*
     Get Country organizations api
     inputs:
-      countryName: String
+      name: String
 
-    Returns the list of organizations associated with given country
+    example query:
+      /api/country-page/country/get-organizations?name=Germany
+
+    Returns the list of organizations associated with country's organizationIDs
 */
 router.get('/get-organizations', (req, res) => {
-  name = req.body.name;
-
-  var query = {name: name};
+  var query = req.query;
   Country.find(query, function(err, country) {
-    if(err) {
+    if(err || country.length == 0) {
       return res.status(countryCodes.getOrganizations.countryNotFound.status).send({
         message: countryCodes.getOrganizations.countryNotFound.message
       });
     }else{
-      return res.status(countryCodes.getOrganizations.success.status).send({
-        organizations: country[0].organizations,
-        message: countryCodes.getOrganizations.success.message
+      var orgs = country[0].organizationIDs;
+      var orgQuery = {_id: {$in: orgs}}; // this queries a list of ids
+      Organization.find(orgQuery, function(err, organizations) {
+        if(err || organizations.length == 0){
+          return res.status(countryCodes.getOrganizations.organizationsNotFound.status).send({
+            message: countryCodes.getOrganizations.organizationsNotFound.message
+          });
+        }else{
+          return res.status(countryCodes.getOrganizations.success.status).send({
+           organizations: organizations,
+           message: countryCodes.getOrganizations.success.message
+          });
+        }
+      })
+    }
+  })
+})
+
+
+/*
+    Insert Organizations API
+    inputs:
+      [{
```

Comment:
The brackets here are unnecessary. Look at how the input comments are formated for the report-trips function